foodji_api/Api/Controllers/UsersDataController.cs

```diff
@@ -0,0 +1,41 @@
+using Application.Command;
+using Application.Dto;
+using Application.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("usersData")]
+public class UsersDataController : ControllerBase
+{
+    private readonly IMediator _mediator;
+    
+    public UsersDataController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    [HttpGet]
+    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<UserDataDto>))]
+    public async Task<IActionResult> GetAllUsersData()
```

Comment:
To be removed or changed for GetUserData (singular).

Reply:
Meh, pas *nécessairement* - ça reste un endpoint de diagnostic / admin, qui ne sera pas accessible à tous une fois l'authorization en place comme il faut, donc ça peut potentiellement rester de quoi d'utile, possiblement.

Reply:
(Exemple qui ne sera **clairement** pas mis en place, mais qui pourrait faire usage de quelque chose de similaire, moyennant de faire une requête intermédiaire à Firebase pour avoir les noms etc. - checher le profil d'un "ami" ou d'un "créateur" à la Ricardo dans un aspect "social media) - si on veut vraiment garder pour ce genre de tests, de dévelopement futur, ou si on veut garder ça "clean" - which is also fair. Sur ça, je me prononce pas, je soulève le point, point.

Reply:
Dans ce cas, il devrait être commented out jusqu'à sécurisation de la route. Pourquoi : et si on oubli ce point d'entrée par la suite?

Reply:
Je vais laisser tel quel pour l'instant, on retirera si c'est deprecated

Reply:
Ya pas d'info sensible ici à s'arracher les cheveux de la tête si c'est pas sécurisé - pas plus que de dire que tu peux accéder à n'importe quelle recette live. C'est Firebase qui gère les infos sensibles - à moins que tu considères sérieusement que le user *id* - qui ne contient *aucune info* - est une valeur sensible. I don't reckon it is.